scripts/labman

```diff
@@ -100,6 +107,73 @@ def config():
                                 db_name, db_user, db_password, db_admin_user,
                                 db_admin_password, log_dir, qiita_server_cert)
 
+@labman.command()
+def patch(verbose=False):
+    """Bring the labman schema up to the latest patch
+
+    Pulls the current patch from the settings table and applies all subsequent
+    patches found in the patches directory.
+    """
+    get_support_file = partial(join, join(dirname(abspath(__file__)), 
+                                          '../labman/db/support_files'))
+    patches_dir = get_support_file('patches')
+
+    # we are going to open and close 2 main transactions; this is a required
+    # change since patch 68.sql where we transition to jsonb for all info
```

Comment:
There might be some confusion at some point down the road, if this comment isn't amended to reference Qiita's patch 68.sql.

Reply:
Good catch!